core/src/test/java/nl/nn/adapterframework/util/MiscTest.java

```diff
@@ -1,43 +1,610 @@
+    }
+
+    @Test
+    public void testResourceToStringForResourceEndOfLineString() throws Exception {
+        URL resource = new URL("http://example.com/");
+        String s1 = Misc.resourceToString(resource, "end of the page");
+        assertEquals(s1, "<!doctype html>end of the page<html>end of the page<head>end of the page    <title>Example Domain</title>end of the pageend of the page    <meta charset=\"utf-8\" />end of the page    <meta http-equiv=\"Content-type\" content=\"text/html; charset=utf-8\" />end of the page    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\" />end of the page    <style type=\"text/css\">end of the page    body {end of the page        background-color: #f0f0f2;end of the page        margin: 0;end of the page        padding: 0;end of the page        font-family: -apple-system, system-ui, BlinkMacSystemFont, \"Segoe UI\", \"Open Sans\", \"Helvetica Neue\", Helvetica, Arial, sans-serif;end of the page        end of the page    }end of the page    div {end of the page        width: 600px;end of the page        margin: 5em auto;end of the page        padding: 2em;end of the page        background-color: #fdfdff;end of the page        border-radius: 0.5em;end of the page        box-shadow: 2px 3px 7px 2px rgba(0,0,0,0.02);end of the page    }end of the page    a:link, a:visited {end of the page        color: #38488f;end of the page        text-decoration: none;end of the page    }end of the page    @media (max-width: 700px) {end of the page        div {end of the page            margin: 0 auto;end of the page            width: auto;end of the page        }end of the page    }end of the page    </style>    end of the page</head>end of the pageend of the page<body>end of the page<div>end of the page    <h1>Example Domain</h1>end of the page    <p>This domain is for use in illustrative examples in documents. You may use thisend of the page    domain in literature without prior coordination or asking for permission.</p>end of the page    <p><a href=\"https://www.iana.org/domains/example\">More information...</a></p>end of the page</div>end of the page</body>end of the page</html>end of the page");
+    }
```

Comment:
In stead of using a big concatinated string, it might be easier and more readable to use `TestFileUtils.getResource` and read the contents from a file instead

Reply:
done